test/load_tests.jl

```diff
@@ -0,0 +1,13 @@
+using Revise
```

Comment:
Copied from the IS PR: Seems like even if the best practice is to install Revise globally, if our tests are going to depend it we ought to have it in `test/Project.toml`

Reply:
remove, we need to remove from IS too.

Reply:
This file is optional. Users don’t have to call it. I say leave it the way it is. The instructions should advise you to install TestEnv.jl and Revise.jl globally.

Reply:
Okay, that works for me